R/clean_names.R

```diff
@@ -1,38 +1,38 @@
 #' @title Cleans names of an object (usually a data.frame).
 #'
 #' @description
-#' Resulting names are unique and consist only of the \code{_} character, numbers, and letters.
-#' Capitalization preferences can be specified using the \code{case} parameter.
+#' Resulting names are unique and consist only of the `_` character, numbers, and letters.
+#' Capitalization preferences can be specified using the `case` parameter.
 #'
 #' Accented characters are transliterated to ASCII.  For example, an "o" with a
 #' German umlaut over it becomes "o", and the Spanish character "enye" becomes
 #' "n".
 #'
 #' This function takes and returns a data.frame, for ease of piping with
-#' \code{`\%>\%`}. For the underlying function that works on a character vector
-#' of names, see \code{\link[janitor]{make_clean_names}}.  \code{clean_names}
-#' relies on the versatile function \code{\link[snakecase]{to_any_case}}, which
+#' ``\%>\%``. For the underlying function that works on a character vector
```

Comment:
Is this right, with two ticks to open and one tick to close?

Reply:
Corrected now 
`%>%` works